tests/test_models.py

```diff
@@ -8,7 +8,7 @@
 from roman_datamodels import stnode
 from roman_datamodels.extensions import DATAMODEL_EXTENSIONS
 
-import util
+from roman_datamodels import test_utils as util
```

Comment:
There is a `roman_datamodels.testing` module that this could go in, that's where I tucked away the factories (modeled after `numpy.testing`)

Reply:
You mean that test_utils.py should go in the testing package? Or the that test_models.py should also go there?

Reply:
The former.  It seems tidy to put all the testing tools into that one `roman_datamodels.testing` namespace (`roman_datamodels.testing.util` in this case).  It's not a big deal though.